app/src/scenes/Menu/Cgu.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import PdfViewer from '../../components/PdfViewer';
+
+const Cgu = () => <PdfViewer source={{ uri: 'https://dashboard-mano.fabrique.social.gouv.fr/cgu.pdf' }} title="Mentions Légales" />;
```

Comment:
CGUs ou Mentions Légales ?

Reply:
De ce que je vois ensuite: 

```suggestion
const Cgu = () => <PdfViewer source={{ uri: 'https://dashboard-mano.fabrique.social.gouv.fr/cgu.pdf' }} title="CGUs" />;
```